lnrpc/routerrpc/router.proto

```diff
@@ -380,6 +380,38 @@ message PairHistory {
     bool last_attempt_successful = 6 [json_name = "last_attempt_successful"];
 }
 
+message BuildRouteRequest {
+    /**
+    The amount to send expressed in msat. If set to -1, the minimum routable
+    amount is used.
```

Comment:
Since 0 can't be used now, can we use that as the default instead? So this would also cover the case of the field not being set.

Reply:
I am a little hesitant to use zero as a magic value for amount, because in other contexts amount can legitimately be zero. Remembering min/max htlc setting. But looking at this particular situation, zero has no meaning. So I changed it.